### PR TITLE
chore: upgrade to v0.6.0

### PR DESCRIPTION
**Pull request by Tirokk**
> [mattwr18](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-03-27T13:00:59Z" title="Friday, March 27th 2020, 2:00:59 pm +01:00">Mar 27, 2020</time>_
_Merged <time datetime="2020-03-27T15:43:22Z" title="Friday, March 27th 2020, 4:43:22 pm +01:00">Mar 27, 2020</time>_
---

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -363,6 +408,14 @@ Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
 - build(deps): bump metascraper-title from 5.10.3 to 5.10.5 in /backend [`#2835`](https://github.com/Human-Connection/Human-Connection/pull/2835)
 - build(deps): bump metascraper-publisher in /backend [`#2836`](https://github.com/Human-Connection/Human-Connection/pull/2836)
 - build(deps): bump metascraper-audio from 5.10.3 to 5.10.5 in /backend [`#2840`](https://github.com/Human-Connection/Human-Connection/pull/2840)
+- Generate changelog with auto-changelog [`02367f9`](https://github.com/Human-Connection/Human-Connection/commit/02367f93e0ad635d5f43adf01695f85f06f4c0d2)
+- clean up UserAvatar a little [`d20421c`](https://github.com/Human-Connection/Human-Connection/commit/d20421cb70c4bda51fdaf59f28c3a060b59bb0cd)
+- display username below slug when there is no date-time [`e93332b`](https://github.com/Human-Connection/Human-Connection/commit/e93332b17361d2c59a841e7c03e93647ec76918f)
+
+#### [0.2.1](https://github.com/Human-Connection/Human-Connection/compare/v0.2.1...0.2.1)
```
Comment (Tirokk) [inline CHANGELOG.md:415]: > [roschaefer](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-03-27T13:21:43Z" title="Friday, March 27th 2020, 2:21:43 pm +01:00">Mar 27, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -363,6 +408,14 @@ Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
 - build(deps): bump metascraper-title from 5.10.3 to 5.10.5 in /backend [`#2835`](https://github.com/Human-Connection/Human-Connection/pull/2835)
 - build(deps): bump metascraper-publisher in /backend [`#2836`](https://github.com/Human-Connection/Human-Connection/pull/2836)
 - build(deps): bump metascraper-audio from 5.10.3 to 5.10.5 in /backend [`#2840`](https://github.com/Human-Connection/Human-Connection/pull/2840)
+- Generate changelog with auto-changelog [`02367f9`](https://github.com/Human-Connection/Human-Connection/commit/02367f93e0ad635d5f43adf01695f85f06f4c0d2)
+- clean up UserAvatar a little [`d20421c`](https://github.com/Human-Connection/Human-Connection/commit/d20421cb70c4bda51fdaf59f28c3a060b59bb0cd)
+- display username below slug when there is no date-time [`e93332b`](https://github.com/Human-Connection/Human-Connection/commit/e93332b17361d2c59a841e7c03e93647ec76918f)
+
+#### [0.2.1](https://github.com/Human-Connection/Human-Connection/compare/v0.2.1...0.2.1)
```


That looks odd

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,10 +4,46 @@ All notable changes to this project will be documented in this file. Dates are d
 
 Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
 
-#### [v0.5.0](https://github.com/Human-Connection/Human-Connection/compare/v0.4.2...v0.5.0)
-
-> 20 March 2020
-
+#### [v0.6.0](https://github.com/Human-Connection/Human-Connection/compare/v0.4.2...v0.6.0)
```
Comment (Tirokk) [inline CHANGELOG.md:7]: > [roschaefer](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-03-27T13:21:59Z" title="Friday, March 27th 2020, 2:21:59 pm +01:00">Mar 27, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -4,10 +4,46 @@ All notable changes to this project will be documented in this file. Dates are d
 
 Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
 
-#### [v0.5.0](https://github.com/Human-Connection/Human-Connection/compare/v0.4.2...v0.5.0)
-
-> 20 March 2020
-
+#### [v0.6.0](https://github.com/Human-Connection/Human-Connection/compare/v0.4.2...v0.6.0)
```


The version intervals look messed up a little

Comment (Tirokk) [reply CHANGELOG.md:7]: > [mattwr18](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-03-27T13:28:20Z" title="Friday, March 27th 2020, 2:28:20 pm +01:00">Mar 27, 2020</time>_

---


I noticed this, and took steps to try and fix it, but was unsuccessful and I didn't know what else to do

**Comment by Tirokk** [comment]
> [cypress[bot]](https://github.com/apps/cypress) **Authored by [cypress[bot]](https://github.com/apps/cypress)**
_<time datetime="2020-03-27T14:33:12Z" title="Friday, March 27th 2020, 3:33:12 pm +01:00">Mar 27, 2020</time>_

---



<p align="center">
  <br />
  
</p>

---

### Test summary
<a href="https://dashboard.cypress.io/projects/qa7fe2/runs/6551/specs"></a> <b title="Passed">66</b>  &bull; <a href="https://dashboard.cypress.io/projects/qa7fe2/runs/6551/failures"></a> <b title="Failed">0</b>  &bull;  <b title="Pending">0</b> &bull;  <b title="Skipped">0</b>

---

### Run details
| | |
| --- | --- |
| **Project** | Human-Connection |
| **Status** | Passed |
| **Commit** | bed6ac5829 <span style="cursor:help;" title="Your CI provider generated this commit by merging this branch into the target branch.">ℹ️</span> |
| **Started** | Mar 27, 2020 2:13 PM |
| **Ended** | Mar 27, 2020 2:33 PM |
| **Duration** | 19:30 <a href="https://on.cypress.io/parallelization" title="Enable parallelization to reduce run duration">💡</a> |
| **OS** | Linux Ubuntu Linux - 16.04 |
| **Browser** | Firefox 68  |

[View run in Cypress Dashboard ➡️](https://dashboard.cypress.io/projects/qa7fe2/runs/6551/specs)



---

This comment has been generated by **cypress-bot** as a result of this project's GitHub integration settings. You can manage this integration in [this project's settings in the Cypress Dashboard](https://dashboard.cypress.io/projects/qa7fe2/settings)

**Comment by Tirokk** [comment]
> [cypress[bot]](https://github.com/apps/cypress) **Authored by [cypress[bot]](https://github.com/apps/cypress)**
_<time datetime="2020-03-27T14:39:09Z" title="Friday, March 27th 2020, 3:39:09 pm +01:00">Mar 27, 2020</time>_

---



<p align="center">
  <br />
  
</p>

---

### Test summary
<a href="https://dashboard.cypress.io/projects/qa7fe2/runs/6552/specs"></a> <b title="Passed">66</b>  &bull; <a href="https://dashboard.cypress.io/projects/qa7fe2/runs/6552/failures"></a> <b title="Failed">0</b>  &bull;  <b title="Pending">0</b> &bull;  <b title="Skipped">0</b>

---

### Run details
| | |
| --- | --- |
| **Project** | Human-Connection |
| **Status** | Passed |
| **Commit** | 29f49a85c0 |
| **Started** | Mar 27, 2020 2:14 PM |
| **Ended** | Mar 27, 2020 2:39 PM |
| **Duration** | 24:26 <a href="https://on.cypress.io/parallelization" title="Enable parallelization to reduce run duration">💡</a> |
| **OS** | Linux Ubuntu Linux - 16.04 |
| **Browser** | Firefox 68  |

[View run in Cypress Dashboard ➡️](https://dashboard.cypress.io/projects/qa7fe2/runs/6552/specs)



---

This comment has been generated by **cypress-bot** as a result of this project's GitHub integration settings. You can manage this integration in [this project's settings in the Cypress Dashboard](https://dashboard.cypress.io/projects/qa7fe2/settings)

**Review by Mogge — APPROVED** [review]
I'm going to approve but see my remarks in the code review.

**Pull request closed without merging**